src/TextBox.cpp

```diff
@@ -14,6 +14,7 @@ TextBox::TextBox(std::unique_ptr<TextSegment>& initialText,
 
   _screenText.push_back(std::move(initialText));
   _screenText.front()->getText().setPosition(_baseRect.left, _baseRect.top);
+  requestDraw();
```

Comment:
Could a name change (just thinking out loud, feel free to disagree) to reDraw() work if we at the same time begin to adopt the this-> formalism?
Then this line would be:
this->reDraw() which seems intuitive.

Reply:
Sure. Although I would probably go with "redraw" (without the capital 'D').